validators/deposit_file.go

```diff
@@ -3,16 +3,16 @@ package validators
 import (
 	"mime/multipart"
 
-	"github.com/sul-dlss-labs/taco/persistence"
+	"github.com/sul-dlss-labs/taco/db"
 )
 
 // DepositFileValidator validates the deposit file request
 type DepositFileValidator struct {
-	repository persistence.Repository
+	repository db.Database
```

Comment:
why do we pass the db.Database value in these?

Reply:
Eventually this validator is going to have to check to see if there are duplicate SourceId or DRUIDs in the repository.